src/backend/distributed/commands/utility_hook.c

```diff
@@ -1489,6 +1482,28 @@ DDLTaskList(Oid relationId, const char *commandString)
 }
 
 
+/*
+ * NontransactionalNodeDDLTask builds a list of tasks to execute a DDL command on a
+ * given target set of nodes with cannotBeExecutedInTransaction is set to make sure
+ * that list is being executed without a transaction.
+ */
+List *
+NontransactionalNodeDDLTask(TargetWorkerSet targets, List *commands)
+{
+	List *ddlJobs = NodeDDLTaskList(NON_COORDINATOR_NODES, commands);
```

Comment:
```suggestion
	List *ddlJobs = NodeDDLTaskList(targets, commands);
```

and callers should pass NON_COORDINATOR_NODES